pkg/commands/build.go

```diff
@@ -0,0 +1,24 @@
+package commands
+
+import (
+	"flag"
+
+	"github.com/hashicorp/actions-go-build/pkg/build"
+	"github.com/hashicorp/composite-action-framework-go/pkg/cli"
+)
+
+type buildOpts struct {
+	buildish
```

Comment:
Coming from the walkthrough, I think this concept may need a comment/explanation here.

Reply:
I added an explanatory comment to the buildish type.